core/webviz/_json_page_element.py

```diff
@@ -9,7 +9,10 @@
 from ._page_element import PageElement
 from builtins import str as text
 
-_json_store_init = StringIO(u'var json_store={};')
+_json_store_init = StringIO(u''.join(['if (json_store === undefined) {',
+                                      '    var json_store={};',
```

Comment:
This change is only necessary if someone wants to collate several page elements without using a page. In that case, any other piece of javascript could be included twice. It would be better to add a __str__ where it ensures that js files are only included once.

Reply:
@eivindjahren with "any other piece of javascript" do you mean the javascript-dependencies for each `PageElement`? I guess/think all pieces of javascript in the `jinja2`-`webviz` `PageElement`-templates only define block-scoped variables?

Could one option be to remove `PageElement.__str__` and only provide `PageElement.get_html(include_jslib=True)`, where the user is explicitly asked if javascript-dependencies are included?